main/res/xml/preferences.xml

```diff
@@ -313,6 +313,7 @@
                         android:text="@string/init_gcvote_password_description" />
 
                     <EditTextPreference
+                        android:enabled="false"
```

Comment:
Did you consider switching it to a simple (and therefore display-only) Preference? I find the disabled look unfitting.
We likely should also adapt the string to tell that this is in fact the geocaching.com username.